packages/ui/components/pages/Fuse/FusePoolPage/SupplyList/AssetSupplyRow.tsx

```diff
@@ -359,7 +360,9 @@ export const AssetSupplyRow = ({
                   Number(utils.formatUnits(asset.supplyBalance, asset.underlyingDecimals)) <
                     UP_LIMIT &&
                   '+'}{' '}
-                {tokenData?.extraData?.shortName ?? tokenData?.symbol ?? asset.underlyingSymbol}
+                {abbreviator(tokenData?.extraData?.shortName, 9) ??
```

Comment:
@peetzweg are we still using `tokenData.extraData.shortname` 
I noticed this only exists in `AssetSupplyRow.tsx` and is omitted in `AssetBorrowRow.tsx`

Let me know if I should remove it or add it to borrow row